lib/internal/streams/destroy.js

```diff
@@ -163,7 +163,6 @@ function isRequest(stream) {
 
 // Normalize destroy for legacy.
 function destroyer(stream, err) {
-  // request.destroy just do .end - .abort is what we want
```

Comment:
Why was this removed?

Reply:
Because it's incorrect, `request.destroy` does not call `.end`.